batch_carolapp/Dockerfile

```diff
@@ -0,0 +1,12 @@
+
+FROM totvslabs/pycarol:2.30.0
+
+WORKDIR /app
+ADD requirements.txt /app/
+ADD batch_carolapp/ /app/batch_carolapp/
```

Comment:
`RUN pip install -r requirements.txt` should be before the `ADD batch_carolapp/ /app/batch_carolapp/`
then if you edit the app files, we dont need to rebuild all the image, just update this.

Reply:
Done. I adjusted the cookie-cutter projects that generated this code.